voxxed-microservices-2018/module-01.adoc

```diff
@@ -222,18 +217,20 @@ You might need to run the command a few times, with different results, as the po
 
 [source,sh]
 $ oc get pods
```

Comment:
`oc get pods -w` might be better in code above so it is vibile the change of pod states

Reply:
Good idea.